File: pset6/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int


while True:
    h = get_int("Height: ")
    if h > 0 and h < 9:
        break

for i in range(0, h, 1):
    for j in range(0, h, 1):
        if i + j < h - 1:
            print(" ", end="")
        else:
            print("#", end="")
    print()

File: pset1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("height: ");

    }

    while (h < 1 || h > 8);

    //For each row
    for (int i = 0; i < h; i++)
    {
        //For each column
        for (int j = 0; j < h; j++)
        {
            if (i + j < h - 1)

            {
                printf(" ");
            }
            else
                //Print a brick
            {
                printf("#");
            }
        }



        //Move to the next row
        printf("\n");
    }


}